### DISCUSSION TITLE
The explorer and the indexer provide different symbols

### DISCUSSION THREAD

**Discussion by vasil-nesterov** [aptos-labs/aptos-developer-discussions#544] [General]
There is a discrepancy in names and symbols of some coins.

A good example is: https://explorer.aptoslabs.com/coin/0x8d87a65ba30e09357fa2edea2c80dbac296e5dec2b18287113500b902942929d::celer_coin_manager::UsdcCoin/info?network=mainnet

The "ceUSDC" symbol you see in the "Symbol" row comes from panora.exchange and it's different from the symbol returned by the indexer - "USDC".

Is there any chance for the indexer to provide the ceUSDC symbol as in the explorer?

**Comment by ying-w** [comment]
Indexer doesn't use off chain (panora) data. It takes symbol as defined by coin creator.

For explorer we decided to use panora because we could have multiple scam coins named APT or USDC and wanted a community agreed upon source of truth

**Reply by vasil-nesterov** [reply]
Thanks man, makes sense.
